### ISSUE TITLE
Skype 8 crashing NVDA and itself

### ISSUE THREAD

**Issue by ironcross32** [nvaccess/nvda#8865]
NVDA crashes hard, a windows error sound plays. NVDA does not restart itself and must be restarted manually. Upon restart, the Skype window is unresponsive and must be closed using the windows dialog for programs in such a state.

### Steps to reproduce:
Open Skype 8, select a group chat and begin to browse the messages with d and shift D for landmarks, as that is a very convenient way to do it in that program.
Crash is intermittent but can usually be made to occur by scrolling back far enough in the message history.
### Actual behavior:
NVDA crashes very hard, an error sound is played and it does not restart. It also does not generate a crash dump.
### Expected behavior:
Users should be able to read as far back in the history as Skype will allow without NVDA or Skype crashing.
### System configuration:
windows 10 X64 on Asus Q505UAR. 1.8Ghz Intel Core i5. 12Gigs RAM and HDD size of 1TB and speed of 5200RPM.
#### NVDA Installed/portable/running from source:
installed
#### NVDA version:
2018.3
#### Windows version:
Version 1809 (OS Build 17763.55)
#### Name and version of other software in use when reproducing the issue:
Skype 8.32.0.53
#### Other information about your system:
system equipped with touchscreen but it is disabled in device manager due to conflicts with UWP version of Skype.
### Other questions:
n/a
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no
[nvda-old.log](https://github.com/nvaccess/nvda/files/2499670/nvda-old.log)

**Comment by ehollig** [comment]
Duplicate of #8813
